pyproject.toml

```diff
@@ -1,66 +1,75 @@
 [build-system]
-requires = ["setuptools>=66", "wheel", "setuptools_scm[toml]>=7.0"]
 build-backend = "setuptools.build_meta"
+requires = ["setuptools>=66", "wheel", "setuptools_scm[toml]>=7.0"]
```

Comment:
Might be good to bump this to at least `78` which removes deprecated ways to specify license.